code/examples/channel/src/app.rs

```diff
@@ -309,3 +311,13 @@ pub async fn run(state: &mut State, channels: &mut Channels<TestContext>) -> eyr
     // We can do nothing but return an error here.
     Err(eyre!("Consensus channel closed unexpectedly"))
 }
+
+fn reload_log_level(_height: Height, round: Round) {
+    use malachitebft_test_cli::logging;
+
+    if round.as_i64() > 0 {
+        logging::reload(logging::LogLevel::Trace);
```

Comment:
Should we make it `Debug`? or add a new config option. I am afraid trace will be too verbose.